src/installing-workflow/configuring-postgres.md

```diff
@@ -0,0 +1,58 @@
+# Configuring Postgres
+
+Deis Workflow's controller component relies on a PostgreSQL database to store platform state.
+
+By default, Deis Workflow ships with the [database] component, which provides an in-cluster PostgreSQL database backed up to in-cluster or off-cluster [object storage]. Currently, for object storage, which is utilized by _several_ Workflow components, only off-cluster solutions such as S3 of GCS are recommended in production environments. Experience has shown that many operators already opting for off-cluster object storage similarly prefer to host Postgres off-cluster as well, using Amazon RDS or similar. When excercising both options, a Workflow installation becomes entirely stateless, and is thus restored or rebuilt with greater ease if the need should ever arise.
+
+## Provisioning off-cluster Postgres
+
+First, provision a PostgreSQL RDBMS using the cloud provider or other infrastructure of your choice. Take care to ensure that security groups or other firewall rules will permit connectivity from your Kubernetes worker nodes, any of which may play host to the Workflow controller component.
+
+Take note of the following:
+
+1. The hostname or public IP of your PostgreSQL RDBMS
+2. The port on which your PostgreSQL RDBMS runs-- typically 5432
+
+Within the off-cluster RDBMS, manually provision the following:
+
+1. A database user (take note of the username and password)
+2. A database owned by that user (take note of its name)
+
+If you are able to log into the RDBMS as a superuser or a user with appropriate permissions, this process will _typically_ look like this:
+
+```
+$ psql -h <host> -p <port> -d postgres -U <"postgres" or your own username>
+> create user <deis username; typically "deis"> with password '<password>';
+> create database <database name; typically "deis"> with owner <deis username>;
+> \q
+```
+
+## Configuring Workflow
+
+The Helm Classic chart for Deis Workflow can be easily configured to connect the Workflow controller component to an off-cluster PostgreSQL database.
+
+* **Step 1:** If you haven't already fetched the Helm Classic chart, do so with `helmc fetch deis/workflow-v2.1.0`
```

Comment:
That's a good question. I think this is the classic conundrum that we have created for ourselves by generating our docs from master / _about_ what is in master. It raises this question every time a new feature is introduced and docs relating to that feature (install docs, for instance) must mention a version number.

Can we LGTM and merge deis/charts#286 and then just sit on this PR for now?